### PR TITLE
Update "3.12 Tag Input" story documentation

### PR DESCRIPTION
**Pull request by milkyfruit**
### 🔦 Summary
Learning to write stories, refactored 3.12 Tags Input story. Updated examples to make them more clear, added descriptions, fixed the header. structure.

### 🔨 Changes to make before merging
- [ ] Add complex examples (Custom Values, Reorder, Styling Options)

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers
- [x] 👨‍🎨 UX change is approved by the Design System UX

- 📚 Change is documented
  - [x] Story
  - API description isn't part of this story
 
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

@mykas

### CODE REVIEW

File: stories/MultiSelect/index.story.js [hunk 1]
```diff
@@ -1,100 +1,151 @@
...
 import LinkTo from '@storybook/addon-links/react';
-import playgroundStoryConfig from '../../src/MultiSelect/docs/MultiSelectPlaygroundConfig';
-import allComponents from '../utils/allComponents';
-
-import ExampleSelectSimpleRaw from '!raw-loader!../../src/MultiSelect/docs/ExampleSelectSimple';
-import ExampleSelectAutocompleteRaw from '!raw-loader!../../src/MultiSelect/docs/ExampleSelectAutocomplete';
-import ExampleSuggestionsRaw from '!raw-loader!../../src/MultiSelect/docs/ExampleSuggestions';
-import ExampleTagInputRaw from '!raw-loader!../../src/MultiSelect/docs/ExampleTagInput';
+import * as examples from './examples';
 import ExampleTagInputSelectionRaw from '!raw-loader!../../src/MultiSelect/docs/ExampleTagInputSelection';
```
Comment (mykas) [inline stories/MultiSelect/index.story.js]: @milkyfruit here, you wanted to have custom styling example but you have used an example that does not have custom styling the dropdown. Which working example would you like to add here?

Comment (milkyfruit) [reply stories/MultiSelect/index.story.js]: image
There was this example that might be useful for the users.

**Comment by milkyfruit** [comment]
@mykas drag and drop example can appear without customSuffix and with already profiled values:
image

**Comment by EgoziE** [comment]
Resolved conflicts.
I see that `RichTextArea`s examples file is missing.
And it's not defined in `stories/index.js`

So I can't merge this PR yet.

**Comment by milkyfruit** [comment]
@simast Looks great!

**Pull request merged**